### ISSUE TITLE
Please publish the source code changes

### ISSUE THREAD

**Issue by SysAdmLS** [hmfaysal/Keychron-Q1-Rotary-Encoder#1]
As already requested on facebook it would be nice if you could share the qmk firmware changes with us so we can build our own firmware or atleast publish the pins the encoder socket uses.

e.g 
#define ENCODERS_PAD_A { B12 }
#define ENCODERS_PAD_B { B13 }

If im not mistaked thats all we really need to build our own firmware as described here : 
https://beta.docs.qmk.fm/using-qmk/hardware-features/feature_encoders

Thanks for your work

**Comment by timmcca-be** [comment]
Here's [my version](https://github.com/timmcca-be/qmk_firmware) - see the latest commit for the changes I made. I put mine on the gtg465x version of the firmware because I wanted the RGB changes from there, but it should be easy to apply the same changes on another version.

**Comment by mgor** [comment]
I've also got the encoder working, thanks to u/sbeedyfreeze on reddit:

https://reddit.com/r/Keychron/comments/pms2dq/_/hgrjsth/?context=1

My version is [here](https://github.com/mgor/qmk_firmware), I kept everything in my "mgor" rev_0102 keymap.

**Comment by timmcca-be** [comment]
@mgor nice to meet you, I'm u/sbeedyfreeze lol

**Comment by bloodf** [comment]
@mgor I looked into your code, and it's all based on the ANSI model right?

**Comment by mgor** [comment]
@bloodf no, it's the iso version. Afaik, [rev 0102](https://github.com/qmk/qmk_firmware/blob/master/keyboards/keychron/q1/rev_0102/readme.md) is iso and [rev 0100](https://github.com/qmk/qmk_firmware/blob/master/keyboards/keychron/q1/rev_0100/readme.md) is ansi.

**Comment by bloodf** [comment]
@mgor Yes, sorry I got it wrong :) got it running now :)

**Comment by kebabhut22** [comment]
Does anyone have any idea how to replicate the browser tab scroll? I've managed to get volume adjustment working but have no idea how to go about coding multiple encoder functions. I've combed through GitHub without finding many resources.
